packages/templates/clients/js/websocket/components/ConnectMethod.js

```diff
@@ -0,0 +1,19 @@
+import { Text } from '@asyncapi/generator-react-sdk';
+import { OnOpen, OnMessage, OnError, OnClose } from './event-handlers';
```

Comment:
```suggestion
import { OnOpen, OnMessage, OnError, OnClose } from './EventHandlerMethods';
```

Here's the correct file path

Reply:
done